ci/update_securityhub_suppressions/suppressions.yml

```diff
@@ -252,6 +264,12 @@ Suppressions:
       notes: >
         These are old GuardDuty findings for resources that no longer exist. At our account level we cannot archive
         them, so we have to suppress the findings here.
+  "guardduty-non-archived-findings-conformance-pack-ddmf23yhn":
+    - resources:
+        - arn:aws:guardduty:eu-west-2:776473272850:finding/00c33c7480de79950ac0d30a2137a504
+      notes: >
+        These are old GuardDuty findings for resources that no longer exist. At our account level we cannot archive
```

Comment:
Why can we not suppress these in guard duty? Or do we not want to?

Reply:
We arten't the organisation account so cannot